std_msgs/Header header
float64[36] stiffness_matrix
float64[36] damping_matrix
float64[36] stiffness_abs_matrix
float64[36] damping_abs_matrix